fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -27,7 +29,12 @@ def self.available_options
                                        description: "The file you want to commit",
                                        is_string: false),
           FastlaneCore::ConfigItem.new(key: :message,
-                                       description: "The commit message that should be used")
+                                       description: "The commit message that should be used"),
+          FastlaneCore::ConfigItem.new(key: :skip_git_hooks,
+                                       description: "Set to true to pass --no-verify to git",
+                                       is_string: false,
```

Comment:
You can also remove this line as `type: Boolean` takes precedence over `is_string` 😊 And `is_string` will soon be removed 💪